index.html

```diff
@@ -52,7 +52,73 @@ <h3 class="stories">Multi-Post Stories</h3>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="aboutme-section">
+    <p class="aboutme-heading">About me</p>
```

Comment:
- [ ] This title "About me" should use a header tag `hx` since it is a heading of a new section and also so that it follows good semantic practices.